### PR TITLE
Fix editorial issues with Architecture Overview section.

### PR DESCRIPTION
**Pull request by msporny**
***
<a href="https://pr-preview.s3.amazonaws.com/w3c/did-core/pull/615.html" title="Last updated on Feb 12, 2021, 5:05 PM UTC (fb22bcb)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/did-core/615/fab20c5...fb22bcb.html" title="Last updated on Feb 12, 2021, 5:05 PM UTC (fb22bcb)">Diff</a>

### CODE REVIEW

**Review by OR13 — APPROVED** [review]

File: index.html [hunk 1]
```diff
@@ -3419,6 +3404,20 @@ <h1>Methods</h1>
...
+method</a> specification is similar to the relationship between the IETF generic
+<a>URI</a> specification ([[RFC3986]]) and a specific <a>URI</a> scheme
+([[IANA-URI-SCHEMES]] (such as the http: and https: schemes specified in
+[[RFC7230]]). It is also similar to the relationship between the IETF generic
+URN specification ([[RFC8141]]) and a specific URN namespace definition (such as
+the <a>UUID</a> URN namespace defined in [[RFC4122]]). The difference is that a
+DID method specification, as well as defining a specific <a>DID scheme</a>, also
+specifies the methods creating, resolving, updating, and deactivating
+<a>DIDs</a> and <a>DID documents</a> using a specific type of <a>verifiable data
+registry</a>.
```
Comment (brentzundel) [inline index.html:3418]:
```suggestion
DID method specification, in addition to defining a specific <a>DID scheme</a>,
also specifies the methods creating, resolving, updating, and deactivating
<a>DIDs</a> and <a>DID documents</a> using a specific type of <a>verifiable data
registry</a>.
```

**Review by brentzundel — APPROVED** [review]
some minor suggestions

File: index.html [hunk 2]
```diff
@@ -529,113 +528,99 @@ <h2>
...
         </dt>
         <dd>
 The subject of a <a>DID</a> is, by definition, the entity identified by the
-<a>DID</a>. The <a>DID subject</a> may also be the <a>DID controller</a>.
-Anything can be the subject of a <a>DID</a>: person, group, organization, physical
-thing, logical thing, etc.
+<a>DID</a>. The <a>DID subject</a> might also be the <a>DID controller</a>.
+Anything can be the subject of a <a>DID</a>: person, group, organization,
+physical thing, or concept. This concept is further defined in <a
+href="#did-subject"></a>.
```
[the remark this answers is not in the document]
Comment (rhiaro) [reply index.html:551]: I don't think I agree with changing concept to idea

**Review by rhiaro — APPROVED** [review]

File: index.html (hunk 2 above)
[the remark this answers is not in the document]
Comment (brentzundel) [reply index.html:551]: What about this?
```suggestion
thing, or concept. This is further defined in <a href="#did-subject"></a>.
```

**Comment by msporny** [comment]
Editorial, multiple reviews, changes requested and made, no objections, merging.

**Pull request merged**